src/azure-cli/azure/cli/command_modules/cosmosdb/_validators.py

```diff
@@ -19,6 +21,19 @@ def validate_ip_range_filter(ns):
         ns.ip_range_filter = ",".join(ns.ip_range_filter)
 
 
+def validate_private_endpoint_connection_id(ns):
+    if ns.connection_id:
+        id_parts = ns.connection_id.split('/')
+        ns.private_endpoint_connection_name = id_parts[-1]
+        ns.account_name = id_parts[-3]
+        ns.resource_group_name = id_parts[-7]
+
```

Comment:
Could you please use a common connection id parser as shown here?
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_validators.py#L1180

Reply:
Thanks for the reference. Fixed this.